README.md

```diff
@@ -1 +1,97 @@
-# hello_microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```

Comment:
- Kindly Remove the `template` logo and if you have a logo that expresses the project you can add other wise leve it empty 👌👌

Reply:
Hey @binhussen 

Thanks for the feedback, This has been fixed and/or removed